gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Job.java

```diff
@@ -53,15 +123,7 @@ public Job(BigQuery bigquery, JobInfo info) {
    * @throws BigQueryException upon failure
    */
   public static Job get(BigQuery bigquery, String job, BigQuery.JobOption... options) {
```

Comment:
Yes I think we can remove this. We should remove such methods also in gcloud-java-resourcemanager